cmake/Modules/OpmSatellites.cmake

```diff
@@ -1,4 +1,8 @@
 # - Build satellites that are dependent of main library
+
+option(ADD_DISABLED_CTESTS "Add the tests which are disabled due to failed preconditions to the ctest output (this makes ctest return an error if such a test is present)" ON)
+mark_as_advanced(ADD_DISABLED_CTESTS)
```

Comment:
you mean making it non-advanced? I think it is a pretty esoteric feature, so it should be hidden by default. That said, if other people want this to be shown by default, I'll budge...